pkg/gatewayserver/io/basicstationlns/basicstationlns.go

```diff
@@ -225,15 +227,20 @@ func (s *srv) handleTraffic(c echo.Context) (err error) {
 		ctx = frequencyplans.WithFallbackID(ctx, fallback)
 	}
 
-	// For gateways with valid EUIs and no auth, we provide the link rights ourselves as in the udp frontend.
 	if auth == "" {
-		ctx = rights.NewContext(ctx, rights.Rights{
-			GatewayRights: map[string]*ttnpb.Rights{
-				uid: {
-					Rights: []ttnpb.Right{ttnpb.RIGHT_GATEWAY_LINK},
+		// If the server allows unauthenticated connections (for local testing), we provide the link rights ourselves as in the udp frontend.
```

Comment:
Don't reference another frontend here

Reply:
Ok but this was there already; https://github.com/TheThingsNetwork/lorawan-stack/blob/a0bec9d2769925b4b44380c343c9e158ef33ae23/pkg/gatewayserver/io/basicstationlns/basicstationlns.go#L228